src/containers/Tools.vue

```diff
@@ -9,12 +9,35 @@
       </div>
     </div>
     <div class="toolsContainer-cards">
+      <card
+        name="Progress Tracker"
+        class="toolsContainer-card"
+        id="progress"
+        :desired-height="100"
+      ></card>
+
       <card name="Contact Your Advisors" class="toolsContainer-card" id="advisors">
         <advisor-card></advisor-card>
       </card>
-      <!--      <card name="Another Card" class="toolsContainer-card" id="x" :desired-width="400">-->
-      <!--        <h1>This one has a smaller width than default!</h1>-->
-      <!--      </card>-->
+
+      <card
+        name="Export Schedule"
+        class="toolsContainer-card"
+        id="export"
+        :desired-width="400"
+        :desired-height="100"
```

Comment:
What's up with the `desired` prefix on the dimensions? Also, can we make these dimensions the default so we don't have to specify them?

Reply:
the default is wider than 400 (see the bigger ones)

Reply:
desired is sorta the preferred. i did this back in the spring. i wanted the tool card framework to be reusable, so i chose to have this as a prop. i could've just had the size grow from the tool card's content, but it would be hard to enforce a preferred width for the overall card.

Reply:
currently, i allow some cards to grow when the screen is resized (not sure if i like this tho). that behavior is really possible because the size of the tool card is regulated from the top down rather than the inner content growing the tool card to be as big as it wants.